src/2d/shallow/landspill/landspill_module.f90

```diff
@@ -0,0 +1,35 @@
+!> @file landspill_module.f90
+!! @brief Top-level module for land-spill simulations. 
+!! @author Pi-Yueh Chuang
+!! @version alpha
+!! @date 2018-09-17
+
+module landspill_module
+    use point_source_collection_module
+    implicit none
+    save
+    public
+
+    !> @brief The state of this module.
+    logical, private:: module_setup = .false.
+
+    !> @brief Object for a collection of point sources.
+    type(PointSourceCollection):: point_sources
+
+contains
+
+    !> @brief Initialize landspill module
+    !! @param[in] point_source_file an optional arg; file for point sources
+    subroutine set_landspill(point_source_file)
+        character(len=*), intent(in), optional:: point_source_file 
+
+        ! open data file for point source collection
```

Comment:
Make sure to set `module_setup = .true.` in this function (once it does something anyway).

Reply:
Done in 61455d9.